### PR TITLE
Graph SQL pumps docs

### PR DESCRIPTION
**Pull request by agata-wit**
## Related Issue (if applicable)

This PR includes the documentation for the GraphQL SQL pumps to the documentation, along with the documentation for the aggregate pump.

It includes sample confs and details about the configuration fields.

[Preview link](https://deploy-preview-2429--tyk-docs.netlify.app/docs/nightly/)

## Types of changes

- [ ] Fixing typo (please merge to production)
- [x] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [x] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/tyk-stack/tyk-pump/tyk-pump-configuration/graph-pump.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+* Size of your records - due to the detailed recording being needed for this Pump’s to function correctly, it is important to note that your records and consequently, your MongoDB storage could increase in size rather quickly.
+* Subgraph requests are not recorded - Requests to tyk-controlled subgraphs from supergraphs in federation setting are currently not recorded by the Graph MongoDB Pump, just the supergraph requests are handled by the Graph MongoDB Pump.
+* UDG requests are recorded but subsequent requests to data sources  are currently ignored.
+* Currently, Graph MongoDB Pump data can not be used in Tyk Dashboard yet, the data is only stored for recording purposes at the moment and can be exported to external tools for further analysis.
+
+{{< tab_end >}}
+{{< tab_start "SQL" >}}
+
+Starting with Version `1.7.0` of Tyk Pump and version `4.3.0` of the Tyk Gateway; It is possible to export GraphQL analytics to an SQL database.
```
Comment (agata-wit) [inline tyk-docs/content/tyk-stack/tyk-pump/tyk-pump-configuration/graph-pump.md]: I think it's Pump 1.8.0 and Tyk 5.0.0 @kofoworola

Comment (kofoworola) [reply tyk-docs/content/tyk-stack/tyk-pump/tyk-pump-configuration/graph-pump.md]: thank you!

**Review by kofoworola — APPROVED** [review]

**Comment by buger** [comment]
I think we need to add links from other GraphQL and Analytics pages to this doc. Otherwise it can be found only via search.

**Comment by agata-wit** [comment]
> I think we need to add links from other GraphQL and Analytics pages to this doc. Otherwise it can be found only via search.

@buger I think this will be important once we start using those analytics in the Dashboard in any way. Right now this is just the Pumps. We don't really have a direct connection between GQL APIs and the logs in Mongo or SQL, meaning we ourselves don't reference those analytics anywhere in our product yet. This will come in the next releases and then I agree - we should link to those pages, now though I don't see how we can do that in a logical way.

**Pull request merged**